### PR TITLE
LIME-101: Labels of required fields are not marked with an asterisk on Log in form

### LINKED ISSUE #101: BUG: Labels of required fields are not marked with an asterisk on Sign in and Sign up form
### Describe your issue

 There is no user-visible marking of mandatory field * after labels of fields Email, Password

### Steps to reproduce

1. Open Login page.
2. Check the labels of fields Email, Password.

### What was the actual result?

Labels Email and Password don't include *

### What was the expected result?

Labels of fields Email, Password are marked with a red * as mandatory

### Put here any screenshots or videos (optional)

image

### What OS are you seeing the problem on?

Windows

### What browsers are you seeing the problem on?

Chrome

### Severity

Minor (S4)

### PR DESCRIPTION
**Pull request by stpalamar**
Added asterisk for input with required properties

### CODE REVIEW

**Review by jhladkov — APPROVED** [review]

**Comment by nikita-remeslov** [comment]
Looks like in design we have gap between input and label
image

**Comment by stpalamar** [comment]
Added label bottom margin and asterisk for sign-up form

**Comment by stpalamar** [comment]
Screenshots
image
image

**Review by dimapopovych — APPROVED** [review]
but why there are changes in package-lock.json?

**Comment by stpalamar** [comment]
> but why there are changes in package-lock.json?

I had this problem with frontend
image

**Review by ivanbilobrov07 — COMMENTED** [review]
Maybe try to copy package.lock from development and paste in your branch and it will work, because there are lots of changes in it and I dont know will it be fine

**Review by anton-otroshchenko — APPROVED** [review]

**Comment by stpalamar** [comment]
> Maybe try to copy package.lock from development and paste in your branch and it will work, because there are lots of changes in it and I dont know will it be fine

I tried but anyway there is same error with module  **@rollup/rollup-win32-x64-msvc**

File: package-lock.json
Comment (nikita-remeslov) [inline package-lock.json:1]: we dont need these changes here

Comment (stpalamar) [reply package-lock.json:1]: done

**Review by nikita-remeslov — APPROVED** [review]

**Pull request merged**